mix.exs

```diff
@@ -20,6 +26,31 @@ defmodule LangfuseSdk.MixProject do
     ]
   end
 
+  defp description do
+    """
+    Langfuse SDK
+    """
+  end
+
+  defp package do
+    [
+      maintainers: ["Thiago Majesk Goulart"],
```

Comment:
@nickgnd, @ArthV, @omaralsoudanii I think this is optional, you guys could either put your name here or we could put something else (Workera?). Leaving this here so we can change it before publishing 👍

Reply:
I figured out the easiest path here was to just removed, so I did 👍